File: python/t2.py
while True:
    try:
        l = int(input())
        for i in range(l):
            d = int(input())
            if d > 4 or d < -9:
                print("Out of data Space!")
            else:
                print("%d %d %d\n%d %d %d" % (d + 5, d + 3, d + 1, d + 4, d + 2, d))
                print("%d %d %d\n%d %d %d" % (d + 5, d + 4, d + 1, d + 3, d + 2, d))
                print("%d %d %d\n%d %d %d" % (d + 5, d + 3, d + 2, d + 4, d + 1, d))
                print("%d %d %d\n%d %d %d" % (d + 5, d + 4, d + 2, d + 3, d + 1, d))
                print("%d %d %d\n%d %d %d" % (d + 5, d + 4, d + 3, d + 2, d + 1, d))

    except Exception as e:
        print(e)

File: t2.c
//
// Created by zx on 2020/4/29.
// 数字排列
#include <stdio.h>

void t2() {
    int l1;
    while (~scanf("%d", &l1)) {
        for (int i = 0; i < l1; i++) {
            int d;
            scanf("%d", &d);
            if (d > 4 || d < -9) {
                puts("Out of data space!");
            } else {
              printf("%d %d %d\n%d %d %d\n" , d + 5, d + 3, d + 1, d + 4, d + 2, d);
              printf("%d %d %d\n%d %d %d\n" , d + 5, d + 4, d + 1, d + 3, d + 2, d);
              printf("%d %d %d\n%d %d %d\n" , d + 5, d + 3, d + 2, d + 4, d + 1, d);
              printf("%d %d %d\n%d %d %d\n" , d + 5, d + 4, d + 2, d + 3, d + 1, d);
              printf("%d %d %d\n%d %d %d\n" , d + 5, d + 4, d + 3, d + 2, d + 1, d);
            }
        }
    }
}